src/main/java/org/folio/circulation/domain/Location.java

```diff
@@ -87,7 +87,7 @@ public String getInstitutionName() {
     return getProperty(institutionRepresentation, "name");
   }
 
-  Location withLibraryRepresentation(JsonObject libraryRepresentation) {
+  public Location withLibraryRepresentation(JsonObject libraryRepresentation) {
```

Comment:
Why has this been made public?

Reply:
residue from first implementation, fixed